composer.json

```diff
@@ -18,7 +18,8 @@
         "php": ">=5.3.2",
         "symfony/framework-bundle": "~2.0",
         "symfony/browser-kit": "~2.0",
-        "doctrine/common": "2.*"
+        "doctrine/common": "2.*",
+        "nelmio/alice": "~1.7"
```

Comment:
this should imho be an optional dependency. in this sense there should maybe then also be an explicit check in the code if the class exists before making an instance.